scss/utilities/_maps.scss

```diff
@@ -81,6 +81,11 @@
 ///   /* #{herman-map-compile($brand-colors, 'color')} */
 @function herman-map-compile($map, $function, $args...) {
   $output: ();
+  $type: type-of($function);
+
+  @if ($type == 'string' and function-exists('get-function')) {
```

Comment:
Because the `$function` value comes from users, they should be responsible for passing in the proper format. If we do it for them, that defeats the purpose of first-class functions. If we're calling this function internally, and we want to upgrade, we should get the function before we pass it in.